truss/templates/base.Dockerfile.jinja

```diff
@@ -0,0 +1,103 @@
+ARG PYVERSION={{config.python_version}}
+
+{% if config.model_framework.value == 'huggingface_transformer' %}
+    {% if config.resources.use_gpu %}
+FROM baseten/baseten-huggingface-transformer-server-gpu-base:latest
```

Comment:
Should we version lock this for consistency?

Reply:
@howespt This part is the same as before. Do you know if the use of the latest tag was to allow us flexibility in updating these images afterwards and have more control that way, without requiring a new baseten_client install on user side. Or do you think we can using specific version here?